pickle/Pickle.java

```diff
@@ -23,8 +23,8 @@ public static void main(String[] args)
     {
         // Create the SymbolTable
         SymbolTable symbolTable = new SymbolTable();
-        
-        try
+        STEntry entryy = symbolTable.getSymbol("ss");
```

Comment:
just gonna leave your garbage tests in a code pull????

Reply:
shit

Reply:
lol

Reply:
garbage